scripts/envhook.py

```diff
@@ -12,10 +12,15 @@ def main(argv):
     parser = argparse.ArgumentParser(description='Install a hook into Python that automatically sources `environment`')
     parser.add_argument('action', choices=['install', 'remove'])
     options = parser.parse_args(argv)
-    import site
-    if hasattr(sys, 'real_prefix'):
-        # virtualenv's site does not have getsitepackages()
-        link_dir = os.path.abspath(os.path.join(os.path.dirname(site.__file__), 'site-packages'))
+    if 'VIRTUAL_ENV' in os.environ:  # Confirm virtual environment is active.
+        import site
+        if hasattr(site, 'getsitepackages'):
+            # If `getsitepackages` is present in site are runningn with `venv` or without a virtual environment.
```

Comment:
Incomprehensible. Typo in `runningn`

Reply:
Consider back-quoting the word site.

Reply:
Both plain Python and `venv` have `getsitepackages()`